docs/UserGuide.md

```diff
@@ -314,8 +335,10 @@ Example:
 Deletes the room at a specified index.
 
 Format: `odel INDEX`
-* `INDEX` refers to the index number shown in the displayed resident list. `INDEX` **must be a positive integer 1,2,3, ...**.
-* `odel` will be blocked if the room is occupied. Run `dealloc` to deallocate the room before attempting to delete the room.
+* `odel` will be blocked if the room is occupied. Run [`dealloc`](#deallocate-resident-from-room-dealloc) to deallocate the room before attempting to delete the room.
```

Comment:
Hmm, if you link up this `dealloc`, then we might have to link up all other command references.

Reply:
Removed